src/main.js

```diff
@@ -3,18 +3,26 @@ var title = document.querySelector('h2');
 var desc1 = document.querySelector('.tagline-1');
 var desc2 = document.querySelector('.tagline-2');
 var image = document.querySelector('.cover-image');
-var btn = document.querySelector('.random-cover-button');
+var priceTag = document.querySelector('.price-tag');
```

Comment:
See if we can use whole section of HTML property for some of this to refactor code